internal/fnruntime/wasmtime.go

```diff
@@ -154,17 +154,33 @@ func (f *WasmtimeFn) Run(r io.Reader, w io.Writer) error {
 	// Try to parse the output as yaml.
 	resourceListOutput, err := yaml.Parse(resultStr)
 	if err != nil {
-		return fmt.Errorf("error parsing output resource list %q: %w", resultStr, err)
+		additionalErrorMessage, err2 := retrieveError(f, resourceList)
```

Comment:
Maybe use a more descriptive name here than `err2`?

Reply:
I gave it another shot.